src/ontology/oba-edit.obo

```diff
@@ -881,22 +876,22 @@ creation_date: 2014-06-04T19:27:15Z
 [Term]
 id: OBA:1000083 ! nucleus length
 synonym: "nucleus length trait" EXACT [GOC:TermGenie]
+relationship: http://purl.org/dc/elements/1.1/contributor http://orcid.org/0000-0002-2908-3327
 created_by: annethessen
```

Comment:
No need to replace `created_by` with `dcterms:contributor` in this ontology?

Reply:
Yes we need to, but we are not there yet :D Thanks for the comment!

Reply:
What about updating the links to HTTPS instead of HTTP?

Reply:
never mind 😄